charts/internal/calico/templates/multus/daemonset-multus-cni.yaml

```diff
@@ -0,0 +1,136 @@
+{{- if .Values.config.multus.enabled }}
+---
+# This manifest installs Multus CNI plugins and network config on
+# each node in a Kubernetes cluster.
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: multus
+  namespace: kube-system
+  labels:
+    k8s-app: multus
+    gardener.cloud/role: system-component
+    node.gardener.cloud/critical-component: "true"
+  {{- if not .Values.autoscaling.staticRequests }}
+  annotations:
+    resources.gardener.cloud/preserve-resources: 'true'
+  {{- end }}
+spec:
+  revisionHistoryLimit: 5
+  selector:
+    matchLabels:
+      k8s-app: multus
+  updateStrategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 2
+  template:
+    metadata:
+      labels:
+        node.gardener.cloud/critical-component: "true"
+        networking.gardener.cloud/to-public-networks: allowed
+        networking.gardener.cloud/to-apiserver: allowed
+        networking.gardener.cloud/to-dns: allowed
```

Comment:
Is this daemonset really talking to the world? I can completely understand the API server and DNS access, but public networks seems surprising. Could you please explain?